jmbase/setup.py

```diff
@@ -10,6 +10,6 @@
       license='GPL',
       packages=['jmbase'],
       install_requires=['twisted==21.7.0', 'service-identity',
-                        'chromalog==1.0.5'],
+                        'chromalog==1.0.5', 'pyln-client'],
```

Comment:
```suggestion
                        'chromalog==1.0.5', 'pyln-client==0.10.2.post0'],
```

Lets pin in to specific version. Looking at release history, there is new release with each c-lightning release, so should be bumped together (post0 is likely some fix after initial upload to pypi.org, they don't allow overwriting files).

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/97cf02c50fc08661e8f47df480a40212e8fc01de